client/src/components/Sub_RelatedItems/SingleProd.jsx

```diff
@@ -92,7 +92,7 @@ const SingleProd = forwardRef(({
   return (
     <div
       ref={ref}
-      className="single-prod container"
+      className={ref ? 'single-prod container focused' : 'single-prod container'}
```

Comment:
nice conditional classname here. I wish I used more of this technique on my widget.